sgpt/handlers/handler.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, role: SystemRole) -> None:
 
     @property
     def printer(self) -> Printer:
-        use_markdown = "APPLY MARKDOWN" in self.role.role
+        use_markdown = "apply markdown" in self.role.role.lower()
```

Comment:
It was intentionally written in uppercase as `"APPLY MARKDOWN"` because we might encounter descriptions such as `"do not apply markdown"` In such cases, we will use markdown formatting for a role that, in fact, does not require markdown.

Reply:
Done.